backend/twitterbot/settings.py

```diff
@@ -1,8 +1,8 @@
 # Quality criteria
 MIN_NB_CONTRIBUTORS = 5
 MIN_NB_RATINGS = 15
-MIN_RELIABILITY_SCORE = 10
-MIN_TOURNESOL_SCORE = 10
+MIN_RELIABILITY_SCORE = 25
```

Comment:
Makes me think the 95% / 5% prior on "should be largely recommended" could be adapted for other criteria. 50/50 would be more appropriate for "entertaining". 

(not a serious suggestion)